src/components/agencyspecific.ts

```diff
@@ -0,0 +1,148 @@
+export function fixHeadsignIcon(headsign: string): string | null {
```

Comment:
headsign should also take null and if so, return null?

Reply:
Not necessary because method is only called when headsign exists. Will update the signature regardless